docs/visual/faq.md

```diff
@@ -27,7 +27,7 @@ For example, if content is loaded based on scrolling down to a particular sectio
 window.scroll(0, 1000);
 ```
 
-### Why is my screenshot not full-page?
+### Why is my screenshot not showing as a full page?
 
 Screener captures a full-page screenshot by default in all browsers. The height of the full-page screenshot is based on the scrollable height of the main document body.
```

Comment:
@kimsauce I'm sorry I may be a bit behind on wether or not we are removing references to Screener in favour of Sauce Visual. I used the latter in my change so wondering if it needs to be changed.

Reply:
I can change that here! The Screener/Sauce Visual change is def something I wanted to address in the docs.

Reply:
Great, if you can make the change, that'd be all. I'll approve once you are ready. Thank you!

Reply:
done!